docs/source/dashboards/index.rst

```diff
@@ -0,0 +1,17 @@
+======================================
+Grafana Dashboards Configuration Guide
+======================================
+
+.. toctree::
+   :maxdepth: 3
+   :hidden:
+
+   Adding and Modifying Dashboards <updating_dashboard>
+   CQL Optimization Dashboard <cql_optimization>
+   Scylla Monitoring Stack Advisor <advisor/index>
+
+There are several dashboards you can use with Scylla Monitoring Stack. Choose a topic to begin:
+
+* :doc:`Adding and Modifying Dashboards <updating_dashboard>`
+* :doc:`CQL Optimization Dashboard <cql_optimization>`
```

Comment:
Why is CQL Optimization Dashboard under "Grafana Dashboards Configuration Guide"?